rlbottraining/common_exercises/kickoff_exercise.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass, field
+from typing import Union
+from pathlib import Path
+from math import pi
+
+from rlbot.utils.game_state_util import GameState, BallState, CarState, Physics, Vector3, Rotator
+from rlbot.matchconfig.match_config import Team, PlayerConfig
+
+from rlbottraining.training_exercise import TrainingExercise, Playlist
+from rlbottraining.grading.grader import Grader
+from rlbottraining.rng import SeededRandomNumberGenerator
+from rlbottraining.common_graders.kickoff_grader import QuickResetKickoffGrader
+
+
+@dataclass
+class KickoffType:
+    pos: Vector3
+    rot: Rotator
+
+
+# Default kickoffs.

```

Comment:
Maybe put these into a class or enum-class such that users of this module only need to import `Spawns` and can go `Spawns.BACK_LEFT`